android/cli/commands/_build.js

```diff
@@ -2817,91 +2819,51 @@ AndroidBuilder.prototype.copyResources = function copyResources(next) {
 		if (!jsFilesToEncrypt.length) {
 			// nothing to encrypt, continue
 			return next();
-		}
-
-		// figure out which titanium prep to run
-		let titaniumPrep = 'titanium_prep';
-		if (process.platform === 'darwin') {
-			titaniumPrep += '.macos';
-		} else if (process.platform === 'win32') {
-			titaniumPrep += '.win32.exe';
-		} else if (process.platform === 'linux') {
-			titaniumPrep += '.linux' + (process.arch === 'x64' ? '64' : '32');
-		}
+		} else if (cloak) {
+			this.logger.info('Encrypting javascript assets...');
+			appc.async.parallel(this,
+				jsFilesToEncrypt.map((file) => {
+					return (done) => {
+						const from = path.join(this.buildAssetsDir, file);
+						const to = path.join(this.buildBinAssetsResourcesDir, file);
+						fs.readFile(from, (error, content) => {
+							if (error) {
+								throw error;
```

Comment:
will throwing the errors inside nested callbacks do what we want? Don't you need to call `done(err)`?